webpack.config.js

```diff
@@ -61,7 +63,7 @@ module.exports = [
             extensions: ['.js']
         },
         target: 'node',
-        externals: { './URLShim': 'URL' },
+        externals: [nodeExternals()],
```

Comment:
How can I verify that this change is correct? The `runner-node.js` script actually checks the files in `src/`, not the Node bundle.

Before you can answer this, my real question is: What is the benefit of having a Node bundle? If I understand correctly, one good reason for using Webpack on Node is transpilation using Babel, but our tests show that this is not needed.

Reply:
> How can I verify that this change is correct? The runner-node.js script actually tests the files in src/, not the Node bundle.

Here is an issue for that https://github.com/Palindrom/Palindrom/issues/245

> Before you can answer this, my real question is: What is the benefit of having a Node bundle? 

The only reason I see is bundling all Palindrom internals (network, noqueue, reconnector etc.) into single module, to reduce friction of resolving them.

Reply:
> The only reason I see is bundling all Palindrom internals (network, noqueue, reconnector etc.) into single module, to reduce friction of resolving them.

I don't think resolving local requires is considered friction in the Node world. However, having an additional step in the build system and the complexity of debugging webpacked code is much more real friction than that.